src/Sylius/Bundle/UserBundle/Command/DemoteUserCommand.php

```diff
@@ -28,7 +28,7 @@ protected function configure()
     {
         $this
             ->setName('sylius:user:demote')
-            ->setDescription('Promotes a user by adding a role.')
+            ->setDescription('Demotes a user by removing a role.')
```

Comment:
yup, it was a typo, from what I understood from #5585. 

to be honest, I haven't tested it locally, but I can do that if required. thow, this line `$user->removeRole($securityRole);` seems pretty clear in what it should do :)